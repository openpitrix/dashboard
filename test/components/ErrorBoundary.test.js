import ErrorBoundary from 'components/ErrorBoundary';

describe('Component ErrorBoundary ', () => {
  it('basic render', () => {
    const wrapper = render(
      <ErrorBoundary>
        <div>error test</div>
      </ErrorBoundary>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
