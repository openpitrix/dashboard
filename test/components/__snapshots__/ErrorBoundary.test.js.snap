// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ErrorBoundary  basic render 1`] = `
<div>
  error test
</div>
`;
